libs/langchain/langchain/retrievers/kendra.py

```diff
@@ -178,6 +180,13 @@ def get_document_attributes_dict(self) -> Dict[str, DocumentAttributeValueType]:
         """Document attributes dict."""
         return {attr.Key: attr.Value.value for attr in (self.DocumentAttributes or [])}
 
+    def get_score_attribute(self) -> str:
+        """Document attributes dict."""
+        if self.ScoreAttributes is not None:
+            return self.ScoreAttributes["ScoreConfidence"]
+        else:
+            return "Low"
```

Comment:
Why do we assume the confidence is "Low" by default?

Reply:
Because "LOW" confidence include all the possible results returned by kendra. In case if there is no "ScoreConfidence" returned by boto3 then we will use "LOW" to prevent code crash.